README.md

```diff
@@ -37,7 +38,10 @@ open_api = OpenAPI(
         )
     },
 )
-print(open_api.json(by_alias=True, exclude_none=True, indent=2))
+if PYDANTIC_V2:
+    print(open_api.model_dump_json(by_alias=True, exclude_none=True, indent=2))
+else:
+    print(open_api.json(by_alias=True, exclude_none=True, indent=2))
```

Comment:
Not sure whether to complicate the README with the compat bits, or just assume v2 going forwards - what do you think?

Reply:
I agree, the conditions complicated the README. I changed the README so it assumes Pydantic 2, but I also added comments for Pydantic 1 users.